source/fundamentals/connection/tuning.txt

```diff
@@ -0,0 +1,145 @@
+==================
+Tuning Connections
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _connection-storms-java:
+
+Overview
+--------
+
+In this guide, you can learn about tuning connections to prevent **connection storms**,
+what connection storms are, and common causes.
+
+A **connection storm** is an event in which application servers
+start an increasing number of connection requests, but the host server
+CPU is unable to schedule them.
+
+.. example::
+
+   Consider an application that uses MongoDB, and is served from an
+   auto scaling service. A sudden spike in traffic could cause the scaling
+   service to stand up many new application instances. This sudden burst
+   in application instances attempting to establish connections to MongoDB
+   is one example of a connection storm.
+
+Common causes for connection storms in MongoDB include:
+
+- Incorrect application behavior
+- Network latency
+- Insufficient hardware for the workload
+- Failover events
+
+In MongoDB, connection storms can have a cascading effect. For example:
```

Comment:
Where did this list come from?

Reply:
https://docs.google.com/document/d/11zoI0jLJBzDmppaBsoTZaE6MWGVFjcmMXCiN3fTp1sE/edit#heading=h.ctqkf9d1o5x4